spec/features/course/homepage_spec.rb

```diff
@@ -6,10 +6,23 @@
 
   with_tenant(:instance) do
     let(:course) { create(:course, :opened) }
-    let(:achievement_feed_notification) do
-      achievement = create(:course_achievement)
+    let(:feed_notifications) do
```

Comment:
Icic. Perhaps this is something we have to agree on - for factories, do we go for the: (i) common case, or the (ii) bare minimal. This was why we were asking about the `course_user` and the approved status, cause that is the common case (except for the case where we want to test user registration and confirmation). 

We're trying to improve it cause I think both of us have spent quite some time fixing factories (at least 2-3 days over the past few PRs) >_<

Maybe we can keep it at the back of our heads first, and figure out if there is a better approach.